# 2D position and orientation of the bounding box center
yolo_service/Pose2D center

# total size of the bounding box, in pixels, surrounding the object's center
yolo_service/Vector2 size